addons/survey/views/survey_templates_statistics.xml

```diff
@@ -144,7 +118,61 @@
+        <div class="card-header">
+            <span class="fa fa-filter"></span>  Filters
+            <span t-if="search_filters or search_finished or search_passed_or_failed" class="float-right clear_survey_filter text-primary"><i class="fa fa-times"></i> Clear All Filters</span>
+        </div>
+        <div class="card-body question_and_page_data o_survey_result p-0">
+            <nav class="navbar navbar-light rounded">
+                <div t-if="question_and_page_data" class="d-flex flex-column flex-sm-row justify-content-between w-100">
+                    <ul class="o_survey_results_topbar_filters nav">
+                        <t t-set="dropdown_item_classes">dropdown-item d-flex align-items-center justify-content-between</t>
+                        <li class="nav-item dropdown mr-2 my-1">
+                            <a href="#" role="button" t-attf-class="btn btn-outline-primary dropdown-toggle #{'active' if search_finished else ''}" data-toggle="dropdown">
+                                <span t-if="search_finished">Completed surveys</span>
+                                <span t-else="">All surveys</span>
+                            </a>
+                            <div class="dropdown-menu">
+                                <a t-attf-class="#{dropdown_item_classes} filter-all #{'active' if not search_finished else ''}"> All surveys
+                                    <span t-if="not search_finished" t-esc="survey_data['count_all']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                                <a t-attf-class="#{dropdown_item_classes} filter-finished #{'active' if search_finished else ''}"> Completed surveys
+                                    <span t-if="not search_finished" t-esc="survey_data['count_finished']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
+                            </div>
+                        </li>
+                        <li t-if="survey.scoring_type != 'no_scoring'" class="nav-item dropdown mr-2 my-1">
+                            <a href="#" role="button" t-attf-class="btn btn-outline-primary dropdown-toggle #{'active' if search_passed_or_failed else ''}" data-toggle="dropdown">
+                                <span t-if="search_failed">Failed surveys only</span>
+                                <span t-elif="search_passed">Passed surveys only</span>
+                                <span t-else="">Passed and failed surveys</span>
+                            </a>
+                            <div class="dropdown-menu">
+                                <a t-attf-class="#{dropdown_item_classes} filter-passed-and-failed #{'active' if not search_passed_or_failed else ''}"> Passed and failed surveys
+                                    <span t-if="not search_passed_or_failed" t-esc="survey_data['count_all']" class="badge badge-pill badge-primary ml-3"/>
+                                </a>
```

Comment:
This adds some long lines + strange spacing between the tag and the text.

Can't we just do:
```xml
<a t-attf-class="#{dropdown_item_classes} filter-passed-and-failed #{'active' if not search_passed_or_failed else ''}">
    <span>Passed and failed surveys</span>
    <span t-if="not search_passed_or_failed" t-esc="survey_data['count_all']" class="badge badge-pill badge-primary ml-3"/>
</a>
```

(Same for other of course)